### DISCUSSION TITLE
Is there a way to set the title of the edit form to a parameter of the current resource/model?

### DISCUSSION THREAD

**Discussion by apvanlaan** [filamentphp/filament#12448] [Help] [answered]
### Package

Form builder

### Package Version

v3

### How can we help you?

Pretty straightforward as the title says.  Say I have a Customer Resource and I want the title of the edit page to be the customer's FullName field, is that possible with Filament?  Thanks in advance

**Comment by leandrocfe** [comment, marked as answer]
Add this in the EditCustomer.php

```php
public function getHeading(): string
{
    return 'Edit: ' . $this->getRecord()->fullname;
}
```

**Reply by apvanlaan** [reply]
Thanks, guess I didn't dig deep enough to find that function, works perfectly

**Comment by t-prod** [comment]
Thanks any idea to add the button on the right to add a model ?
And also the edit button opens a modal I wold like to have only a real edit button with the route helper ;)

**Reply by leandrocfe** [reply]
Please open a new topic and explain what you are trying to achieve
